bramkouwen/Neatly-In-Motion
# Adding Project Folders

## What does this PR do?
* It creates the project folders

## How do you check this PR?
* Pull down the branch
* Check the added project folders

## Screenshots 
None.
